### PR TITLE
Cade buffs, queen eye slight nerf

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

Makes cades cheaper, adds a new kind of metal folding cade, and makes it harder to chungus past cadelines with the queen eye. Xenos gotta damage them cades first.

## Why It's Good For The Game

Makes cades a bit stronker and lets marines get some more of them, also ends the tyranny of plas on req.

## Changelog

:cl:
balance: Reduced the metal required to build metal cades from 5 to 4.
balance: Reduced the metal required to build barbed wire from 2 to 1.
balance: Reduced the plasteel required to build plasteel cades from 10 to 8.
balance: Weeds can no longer spread onto tiles with barricades on them from the side that the barricade is protecting if the health of the barricade is over 25%.
add: Added metal folding barricades, 400 HP, can be flipped like plasteel cades but cannot be linked. 6 metal to build. Sprites by Triiodine.
/:cl:

### CODE REVIEW

**Comment by PhantomEpicness** [comment]
based

**Review by Nanu308 — APPROVED** [review]
Balance Wise Fine

File: code/modules/cm_aliens/weeds.dm [hunk 1]
```diff
@@ -199,12 +199,20 @@
 	for(var/obj/structure/platform/P in src.loc)
 		if(P.dir == reverse_direction(direction))
 			return FALSE
+	for(var/obj/structure/barricade/B in loc)
+		if(B.density && B.dir == direction && B.health >= (B.maxhealth / 4))
+			return FALSE
```
Comment (Watermelon914) [inline code/modules/cm_aliens/weeds.dm:204]: rename B to a more descriptive variable name, 'barricade' if nothing else.

**Review by Watermelon914 — CHANGES_REQUESTED** [review]

File: code/modules/cm_aliens/weeds.dm (hunk 1 above)
Comment (stanalbatross) [reply code/modules/cm_aliens/weeds.dm]: done

**Review by Shad0vvs — APPROVED** [review]
Changes done, Sprites were approved already on gitlab.

**Pull request merged**